hardware/arduino/sam/libraries/Wire/Wire.cpp

```diff
@@ -408,7 +433,21 @@ static void Wire1_Init(void) {
 	NVIC_EnableIRQ(WIRE1_ISR_ID);
 }
 
-TwoWire Wire1 = TwoWire(WIRE1_INTERFACE, Wire1_Init);
+static void Wire1_Deinit(void) {
+	NVIC_DisableIRQ(WIRE1_ISR_ID);
+	NVIC_ClearPendingIRQ(WIRE1_ISR_ID);
+
+	pmc_disable_periph_clk(WIRE1_INTERFACE_ID);
+
+	// disable pull ups
+	pinMode(PIN_WIRE1_SDA, OUTPUT);
+	pinMode(PIN_WIRE1_SCL, OUTPUT);
```

Comment:
@cmaglie upon further investigation, this code isn't needed, as the ``PIO_Configure`` calls  in the ```Wire1_Init``` do not enable pull ups. I've updated this in: https://github.com/sandeepmistry/Arduino/commit/dd7e90802e960904f281c8fb6ac24b7fae29e863

I was confused my the external SDA0 and SCL0 pull ups on the Due.